docs/using-hive.md

```diff
@@ -15,7 +15,7 @@
         ```bash
         oc create secret generic mycluster-ssh-key --from-file=ssh-privatekey=/path/to/private/key --from-file=ssh-publickey=/path/to/public/key
         ```
-     **NOTE**: This step is optional. This will be done automatically if using `hiveutil create-cluster` with `--ssh-public-key-file` and `--ssh-private-key-file` arguments. 
+     **NOTE**: This step is optional. This will be done automatically if using `hiveutil create-cluster` with `--ssh-public-key-file` and `--ssh-private-key-file` arguments.
```

Comment:
this section is gone in #823

Reply:
My editor is setup to trim trailing whitespace so it's picking this up on it's own. :( Will deal with it on merge conflict.